config/twitter_keys.EXAMPLE.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    consumer_key: '',
+    consumer_secret: '',
```

Comment:
1. do you need the consumer key and secret? back in the day you could get away with just the access_token_key and access_token_secret similar to what I wrote in the Github wrapper in libs?

2. Might be good to do it like that to be consistent. I made it so it reads either a config passed in through the CLI or a ENV variable (`process.env.NAME_OF_VAR`)